.circleci/config.yml

```diff
@@ -20,6 +20,8 @@ jobs:
             INSTALL_SMALL_PYTHON_DEPS: true
             INSTALL_LARGE_PYTHON_DEPS: true
           command: |
+            pip --version
```

Comment:
This prints out the following on CircleCI:

```
pip 20.3 from /usr/local/lib/python3.6/site-packages/pip (python 3.6)
```

Reply:
![Screen Shot 2020-12-03 at 11 15 40](https://user-images.githubusercontent.com/17039389/100954924-e9bffe00-3558-11eb-9268-17da1be702ba.png)